tools/test-proxy/Azure.Sdk.Tools.TestProxy/Store/IAssetsStore.cs

```diff
@@ -21,6 +22,7 @@ public interface IAssetsStore
         /// Given a configuration, determine the state of the resources present under contextPath, reset those resources to their "fresh" state.
         /// </summary>
         /// <param name="pathToAssetsJson"></param>
-        public abstract Task Reset(string pathToAssetsJson);
+        /// <param name="consoleWrapper"></param>
+        public abstract Task Reset(string pathToAssetsJson, IConsoleWrapper consoleWrapper);
```

Comment:
IConsoleWrapper should move to constructor for `GitStore`.

Reply:
fixed with next commit